component-samples/demo/rv/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
     container_name: pri-fdo-rv
     expose:
       - "8040"
-      - "8041"
+     - "8041"
```

Comment:
indentation

Reply:
fixed in commit ac5c71922b41df4598d1eb9b068ce0a5edfe89fc